README.md

```diff
@@ -206,7 +193,32 @@ wiki page.
    Additionally, the cause of the "PCM not found" error might be an incorrect ALSA PCM name. Run
    `bluealsa-aplay --list-pcms` for the list of currently available ALSA PCM names - it might give
    you a hint what is wrong with your `.asoundrc` entry. Also, take a look at the
-   [bluealsa-plugins man page](./doc/bluealsa-plugins.7.rst).
+   [bluealsa-plugins manual page](./doc/bluealsa-plugins.7.rst).
+
+## Contributing
+
+This project welcomes contributions of code, documentation and testing. For code and manual page contributions, please use Github Pull Requests. There is no strict policy for PRs, each contribution will be evaluated individually. If you wish to help by testing PRs or by making review comments please do so by adding comments to the PR.
+
+When preparing a pull request, if possible please configure with `--enable-test`, and to catch as many coding errors as possible please compile with:
+```
+make CFLAGS="-Wall -Wextra -Werror"
+```
+and then run the unit test suite:
+```
+make check
+```
+
+The project wiki is "public" and contributions there are also welcome.
+
+## Bug reports, feature requests, and requests for help
+
+Please search previous issues (both open and closed), to see if your question has already been answered or problem resolved, before raising a new issue. If reporting a problem, please clearly state the version of BlueALSA that you are using, and give sufficient information for readers to be able to reproduce the issue.
```

Comment:
I'm not linguist in any way, but maybe something like this will be better:

```suggestion
Before raising a new issue, please search previous issues (both open and closed), to see if your question has already been answered or problem resolved. If reporting a problem, please clearly state the version of BlueALSA that you are using, and give sufficient information for readers to be able to reproduce the issue.
```